pkg/dhcp6client/dhcp6client.go

```diff
@@ -92,11 +107,17 @@ func (p *buffer) remaining() []byte {
 
 func (c *Client) ReadReply() (*dhcp6.Packet, error) {
 	var err error
-	for i := 0; i < 5; i++ { // five attempts
+	for i := 0; i < 5; i++ {
```

Comment:
We can make it 5 in default and set a flag of the amount of attempts. What do you think?